src/helpers/interfaces.ts

```diff
@@ -0,0 +1,18 @@
+export interface Metadata {
+  ci: {
+    pipeline: {
+      url?: string
+    }
+    provider: {
+      name: string
+    }
+  }
+  git: {
+    branch?: string
+    commit_sha?: string
```

Comment:
Inconsistent case, let's rename this one `commitSha`. It's a breaking change but it's ok since it was never used anywhere in Synthetics UI *but* it still requires a change of our schemas so it doesn't break the intake -> https://datadoghq.atlassian.net/browse/SSV-252

Reply:
ok so to confirm this is blocked until https://datadoghq.atlassian.net/browse/SSV-252 is done, right?

Reply:
ok yeah tests are failing: https://github.com/DataDog/datadog-ci/runs/1320388975?check_suite_focus=true

Reply:
Shouldn't this be done in a separate PR @sdeprez ?

Reply:
that makes sense. If @sdeprez agrees I'll separate this into a different PR

Reply:
Whatever works yes, it can be done in another PR to unlock this one (as long as both PRs are done before someones uses this new command so it has proper naming `commitSha` since the beginning!)